CMakeLists.txt

```
@@ -115,9 +126,16 @@ endif ( )
 #
 # Install
 #
-file( GLOB ARTIFACTS "${SOURCE_DIR}/*.hpp" )
+if ( BUILD_DEVEL_PACKAGE )
+    file( GLOB ARTIFACTS "${SOURCE_DIR}/*.hpp" )
+    install( FILES "${INCLUDE_DIR}/${PROJECT_NAME}" DESTINATION "${CMAKE_INSTALL_PREFIX}/include" )
+    install( FILES ${ARTIFACTS} DESTINATION "${CMAKE_INSTALL_PREFIX}/include/corvusoft/${PROJECT_NAME}" )
+endif ( )
 
-install( FILES "${INCLUDE_DIR}/${PROJECT_NAME}" DESTINATION "${CMAKE_INSTALL_PREFIX}/include" )
-install( FILES ${ARTIFACTS} DESTINATION "${CMAKE_INSTALL_PREFIX}/include/corvusoft/${PROJECT_NAME}" )
-install( TARGETS ${STATIC_LIBRARY_NAME} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT library )
-install( TARGETS ${SHARED_LIBRARY_NAME} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT library )
+if ( BUILD_STATIC_LIBRARY )
+    install( TARGETS ${STATIC_LIBRARY_NAME} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT library )
+endif ( )
+
+if ( BUILD_SHARED_LIBRARY )
+    install( TARGETS ${SHARED_LIBRARY_NAME} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT library )
```

You need to install `restbed.dll` too.
```suggestion
    install( TARGETS ${SHARED_LIBRARY_NAME} LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR} ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT library RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR} COMPONENT runtime)
```

---

What does this proposed change has to do with my Pull Request? I just have indented this already existing line because I have surrounded it with if( <switch> )..endif().

If there is something to improve for the MS-Windows build, I suggest making a separate Pull Request.